packages/react-core/src/components/Select/Select.tsx

```diff
@@ -146,6 +146,8 @@ export interface SelectProps
   chipGroupComponent?: React.ReactNode;
   /** Flag for retaining keyboard-entered value in typeahead text field when focus leaves input away */
   isInputValuePersisted?: boolean;
+  /** Flag for retaining filter results on blur from keyboard-entered typeahead text applied with isInputValuePersisted */
```

Comment:
Can you add `@beta` at the beginning of this prop description?

Reply:
Sure no problem